entrypoint/fb.js

```diff
@@ -165,27 +166,37 @@ const handleTextMessage = async (chat) => {
     // React to intent
     switch (lexResponse.intentName) {
         case 'stop':
+            if (chat.trackingEnabled) {
+                await chat.track.event('Conversation-ShortMessage', 'Unsubscribe', chat.language).send();
+            }
             return unsubscribe(chat);
         case 'help':
+            if (chat.trackingEnabled) {
+                await chat.track.event('Conversation-ShortMessage', 'SubscriptionHelpVideo', chat.language).send();
+            }
             return subscriptionHelp(chat);
     }
 
-    let textReply;
-    console.log(lexResponse);
-    switch (lexResponse.messageFormat) {
-        case 'Composite':
-            const groups = JSON.parse(lexResponse.message);
-            textReply = groups.messages[Math.floor(Math.random() * groups.messages.length)].value;
-            break;
-        case 'PlainText':
-        case 'CustomPayload':
-            textReply = lexResponse.message;
+    if (chat.trackingEnabled) {
+        await chat.track.event('Conversation-ShortMessage', lexResponse.intentName, chat.language).send();
     }
 
-    if (chat.trackingEnabled) {
-        await chat.track.event('Conversation', lexResponse.intentName, chat.language).send();
+    let textReply;
+    if (lexResponse.message === null) {
```

Comment:
Did you test this? If you expect there is no message at all, you should compare to `undefined`

```$ node
> const foo = {}
undefined
> foo.message === null
false
> foo.message === undefined
true
```

Reply:
sorry, it is indeed set to `null` explicitly by the library